Regression/WarpX-tests.ini

```diff
@@ -3030,6 +3030,24 @@ compareParticles = 1
 particleTypes = electrons he_ions
 analysisRoutine = Examples/analysis_default_regression.py
 
+[background_mcc_dp_psp]
+buildDir = .
+inputFile = Examples/Physics_applications/capacitive_discharge/inputs_2d
+runtime_params = warpx.abort_on_warning_threshold = high
+dim = 2
+addToCompileString =
```

Comment:
@ax3l Right now, this will build in the 2d pipeline. Do we want to adjust the test selection in `prepare_file_ci.py` to encompass this mixed-precision test?

More broadly, looking at this, following my recent CI adjustments in #3107 (still to be merged), I think it may be worth amending that script to strictly partition the tests, to ensure tests are neither doubled nor dropped on the floor.

Reply:
That sounds reasonable to me.

Generally speaking, we try to have the number of unique binary builds per Azure entry <=3, IIRC. That way, a cache re-silvering (usually on Mondays after we merge the AMReX weekly update) can pass within an hour.

Reply:
@KZhu-ME @PhilMiller as a follow-up PR, can we add a test that does the exact opposite: compiling in `SP` for `amrex::Real` but using `DP` for `amrex::ParticleReal`? :)

Reply:
Sounds good, I'm trying to run the `background_mcc` test in that configuration with `SP` for `Real` and `DP` for `ParticleReal`, but I'm running into particle out of bounds errors on `RedistributeCPU` calls.

Reply:
Sounds good - let's debug in the follow-up PR.